I found a problem while unpacking a apk named by “xxx  .apk” --- with the blank space at the end of the apk name.

hi, I found a problem while unpacking a apk named by “xxx  .apk” --- with the blank space at the end of the apk name.   (apktool V2.0.3  on Win7 )

```
F:\test>apktool.bat d -f "test  .apk"
I: Using Apktool 2.1.0-a64a03-SNAPSHOT on test  .apk
I: Loading resource table...
I: Decoding AndroidManifest.xml with resources...
Exception in thread "main" brut.androlib.AndrolibException: brut.directory.Direc
toryException: java.io.FileNotFoundException: test  \AndroidManifest.xml (系统找
不到指定的路径。)
        at brut.androlib.res.decoder.ResFileDecoder.decodeManifest(ResFileDecode
r.java:142)
        at brut.androlib.res.AndrolibResources.decodeManifestWithResources(Andro
libResources.java:199)
        at brut.androlib.Androlib.decodeManifestWithResources(Androlib.java:142)

        at brut.androlib.ApkDecoder.decode(ApkDecoder.java:100)
        at brut.apktool.Main.cmdDecode(Main.java:165)
        at brut.apktool.Main.main(Main.java:81)
Caused by: brut.directory.DirectoryException: java.io.FileNotFoundException: tes
t  \AndroidManifest.xml (系统找不到指定的路径。)
        at brut.directory.FileDirectory.getFileOutputLocal(FileDirectory.java:64
)
        at brut.directory.AbstractDirectory.getFileOutput(AbstractDirectory.java
:115)
        at brut.androlib.res.decoder.ResFileDecoder.decodeManifest(ResFileDecode
r.java:138)
        ... 5 more
Caused by: java.io.FileNotFoundException: test  \AndroidManifest.xml (系统找不到
指定的路径。)
        at java.io.FileOutputStream.open0(Native Method)
        at java.io.FileOutputStream.open(Unknown Source)
        at java.io.FileOutputStream.<init>(Unknown Source)
        at java.io.FileOutputStream.<init>(Unknown Source)
        at brut.directory.FileDirectory.getFileOutputLocal(FileDirectory.java:62
)
        ... 7 more
```
************************************************************************************************************************
But, It is OK with the param "-o"

F:\test>apktool.bat d -f "test  .apk" -o test

so,  Apktool do not handle the path correctly ?

--- a/brut.apktool/apktool-cli/src/main/java/brut/apktool/Main.java
+++ b/brut.apktool/apktool-cli/src/main/java/brut/apktool/Main.java
@@ -151,7 +151,7 @@ public class Main {
             // make out folder manually using name of apk
             String outName = apkName;
             outName = outName.endsWith(".apk") ? outName.substring(0,
-                    outName.length() - 4) : outName + ".out";
+                    outName.length() - 4).trim() : outName + ".out";
 
             // make file from path
             outName = new File(outName).getName();

